src/Compute/Compute/Generated/ResourceSku/ResourceSkuListMethod.cs

```diff
@@ -36,13 +36,25 @@ namespace Microsoft.Azure.Commands.Compute.Automation
     [OutputType(typeof(PSResourceSku))]
     public partial class GetAzureRmComputeResourceSku : ComputeAutomationBaseCmdlet
     {
+        [Parameter(Mandatory = false, Position = 0, ValueFromPipelineByPropertyName = true), ValidateNotNullOrEmpty, LocationCompleter("Microsoft.Compute/locations/publishers")]
```

Comment:
NIT: Could you change it to multiple lines? (to make it consistent with other parameters)